debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -112,6 +112,68 @@ public static HStoreHandlingMode parse(String value, String defaultValue) {
         }
     }
 
+    /**
+     * Defines modes of representation of {@code interval} datatype
+     */
+    public enum IntervalHandlingMode implements EnumeratedValue {
+
+        /**
+         * Represents interval as inexact microseconds count
+         */
+        MICROSECONDS("microseconds"),
```

Comment:
For Cassandra, nanoseconds are used. Could we name it "numeric" or something similar, so we can use the same option over there, too?

Reply:
Speaking of which, could you do the Cassandra change, too?

Reply:
@gunnarmorling Unfortunately I can't. Please see the follow-up Jiras
https://issues.jboss.org/browse/DBZ-1537
https://issues.jboss.org/browse/DBZ-1538

Reply:
Ok, thanks. Pinged @bingqinzhou about it, too, in case he's interested in picking up this one.